src/fibonacci_n-step_number_sequences.rs

```diff
@@ -0,0 +1,68 @@
+// http://rosettacode.org/wiki/Fibonacci_n-step_number_sequences
+
```

Comment:
You need a `#![allow(unstable)]` here to avoid warnings related to the usage of unstable items.